railgun/pkg/config/config.go

```diff
@@ -37,6 +38,7 @@ type Config struct {
 	KongWorkspace      string
 	AnonymousReports   bool
 	EnableReverseSync  bool
+	SyncPeriod         time.Duration
```

Comment:
would improve code maintainability if a note "sync in Second, generically" or Millisecond (possibly) is added

Reply:
Generally speaking, this will never be in milliseconds this will be in hours. A comment was added to this effect in the flag declaration:

https://github.com/Kong/kubernetes-ingress-controller/pull/1506/files#diff-777c5b378220515f6b692537382429104081d22a761062c7e2f0919e6c0d1757R118

Is this sufficient or do you want something more?